code/modules/status_system/statusEffects.dm

```diff
@@ -2751,3 +2751,53 @@
 			var/mob/living/M = src.owner
 			M.bioHolder.RemoveEffectInstance(src.added_accent)
 		UnregisterSignal(src.owner, COMSIG_MOB_SAY)
+
+/datum/statusEffect/graffiti
+	id = "graffiti"
```

Comment:
Could probably change id to something more descriptive than "graffiti"? Considering this is used on mobs, specifically for blocking their vision